exporters/trace/gcp_exporter/internal/recordable.cc

```diff
@@ -43,6 +42,22 @@ void Recordable::SetAttribute(nostd::string_view key,
     {
         (*map)[std::string(key)].set_bool_value(nostd::get<bool>(value));
     } 
+    else if (nostd::holds_alternative<int>(value))
+    {
+        (*map)[std::string(key)].set_int_value(nostd::get<int>(value));
+    }
+    else if (nostd::holds_alternative<int64_t>(value))
+    {
+        (*map)[std::string(key)].set_int_value(nostd::get<int64_t>(value));
+    }
+    else if (nostd::holds_alternative<unsigned int>(value))
+    {
+        (*map)[std::string(key)].set_int_value(nostd::get<unsigned int>(value));
+    }
+    else if (nostd::holds_alternative<uint64_t>(value))
+    {
+        (*map)[std::string(key)].set_int_value(nostd::get<uint64_t>(value));
+    }
```

Comment:
Is there any plan to support array attribute types? Or does GCP not support that?

Reply:
[Official docs](https://cloud.google.com/trace/docs/reference/v2/rpc/google.devtools.cloudtrace.v2#google.devtools.cloudtrace.v2.Span.Attributes) say it only takes in three types: bool, int, and string.